decayAmplitude/massDependence.cc

```diff
@@ -564,6 +617,231 @@ piPiSWaveAuMorganPenningtonKachaev::piPiSWaveAuMorganPenningtonKachaev()
 }
 
 
+////////////////////////////////////////////////////////////////////////////////
+KPiSGLASS::KPiSGLASS(const double a, const double r, const double M0, const double G0, const double phiF, const double phiR, const double phiRsin,
+                     const double F, const double R, const double MMax, const std::string& tag)
```

Comment:
Give equation number in paper.
It would be nice to briefly explain meaning of parameters, if possible

Reply:
Done in the header file